### ISSUE TITLE
Bug: Erroneous minus operator in libc streamlist.c

### ISSUE THREAD

**Issue by KyleCardoza** [hth313/Calypsi-tool-chains#20]
Lines 40 through 50 of streamlist.c:

```C
  _stderr.fs_fd = 2;
  _stderr.fs_oflags = O_WRONLY;

  __global_streams.sl_std[0] = &_stdin;
  __global_streams.sl_std[1] = &_stdout;
  __global_streams.sl_std[2] - &_stderr;

  // Register for having any open stream closed by _Exit() termination
  _ExitCloseStreams = closeAllStreams;
}
```

Note the third assignment is erroneously a (null effect) subtract operation, when it is clearly meant to be an assignment.

**Comment by hth313** [comment]
Ouch, fixed for the next release 4.4.
I released 4.3 today and this sadly came just afterwards.

**Comment by KyleCardoza** [comment]
I had meant to report it a few days ago and it got away from me. Apologies. I only found it because I build with -Wall -Wextra -Wpedantic -Werror by default, disabling specific warnings as needed.
